bool rerun 
string path #Used only on rerun mode
uint16 n_movable_object 
uint16 n_vase
bool randomized_vase #both the pose and the shape
---
